openrtb2/video.go

```diff
@@ -112,8 +187,9 @@ type Video struct {
 	//   where 0 = no, 1 = yes.
 	//   If a bidder sends markup/creative that is itself skippable, the
 	//   Bid object should include the attr array with an element of
-	//   16 indicating skippable video. Refer to List 5.3.
-	Skip *int8 `json:"skip,omitempty"`
+	//   16 indicating skippable video. Refer to List: Creative
+	//   Attributes in AdCOM 1.0.
+	Skip *adcom1.CreativeAttribute `json:"skip,omitempty"`
```

Comment:
TODO: adcom1 type used by mistake. This property can hold only 1/0 (int8/bool). CreativeAttribute is mentioned for the "attr" prop.